app/components/form-command/template.hbs

```diff
@@ -122,16 +154,39 @@
   </div>
 
   <div class="col-sm-12 col-md-8">
-    {{form-key-value
-      initialMap=instance.environment
-      changed=(action (mut instance.environment))
-      addActionLabel="formCommand.environment.addAction"
-      keyLabel="formCommand.environment.keyLabel"
-      keyPlaceholder="formCommand.environment.keyPlaceholder"
-      valueLabel="formCommand.environment.valueLabel"
-      valuePlaceholder="formCommand.environment.valuePlaceholder"
-      allowEmptyValue=true
-    }}
+    {{#if editing}}
+      {{form-key-value
+        initialMap=instance.environment
+        changed=(action (mut instance.environment))
+        addActionLabel="formCommand.environment.addAction"
+        keyLabel="formCommand.environment.keyLabel"
+        keyPlaceholder="formCommand.environment.keyPlaceholder"
+        valueLabel="formCommand.environment.valueLabel"
+        valuePlaceholder="formCommand.environment.valuePlaceholder"
+        allowEmptyValue=true
+      }}
+    {{else}}
+      {{#if instance.environment}}
+        <table class="grid fixed">
+          <thead>
+            <tr>
+              <th>{{t 'formCommand.table.key'}}</th>
```

Comment:
Reusing `formCommand.environment.keyLabel` and `valueLabel` seems fine here.. duplicate strings are not fun to translate.